From: Marco d'Itri <md@linux.it>
Date: Wed, 25 Jan 2012 19:14:26 +0100
Subject: [PATCH] fix_warnings

fix minor issues

Fix warnings, missing prototypes, an incomplete clean target.
---
 pppd/ipv6cp.c |    2 +-
 pppd/pppd.h   |    2 ++
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/pppd/ipv6cp.c b/pppd/ipv6cp.c
index 4a09c9a..d01b778 100644
--- a/pppd/ipv6cp.c
+++ b/pppd/ipv6cp.c
@@ -1120,7 +1120,7 @@ ipv6_check_options()
 
     if (demand && (eui64_iszero(wo->ourid) || eui64_iszero(wo->hisid))) {
 	option_error("local/remote LL address required for demand-dialling\n");
-	exit(1);
+	die(1);
     }
 }
 
diff --git a/pppd/pppd.h b/pppd/pppd.h
index cf9840a..87522d0 100644
--- a/pppd/pppd.h
+++ b/pppd/pppd.h
@@ -642,6 +642,8 @@ int  sif6addr __P((int, eui64_t, eui64_t));
 				/* Configure IPv6 addresses for i/f */
 int  cif6addr __P((int, eui64_t, eui64_t));
 				/* Remove an IPv6 address from i/f */
+int  ether_to_eui64 __P((eui64_t *));
+				/* Convert Ethernet address into 64-bit EUI */
 #endif
 int  sifdefaultroute __P((int, u_int32_t, u_int32_t));
 				/* Create default route through i/f */
